src/Appwrite/Utopia/Response/Model/AttributeRelationship.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+
+class AttributeRelationship extends Attribute
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this
+            ->addRule('relatedCollection', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The Id of the related collection',
+                'default' => null,
+                'example' => 'collection',
+            ])
+            ->addRule('relationType', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The type of the relationship',
+                'default' => '',
+                'example' => 'oneToOne|oneToMany|manyToOne|manyToMany',
+            ])
+            ->addRule('twoWay', [
+                'type' => self::TYPE_BOOLEAN,
+                'description' => 'Is the relationship two-way?',
+                'default' => false,
+                'example' => false,
+            ])
+            ->addRule('twoWayKey', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The key of the two-way relationship',
+                'default' => '',
+                'example' => 'string',
+            ])
+            ->addRule('onDelete', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Action to take on related documents when parent document is deleted',
```

Comment:
```suggestion
                'description' => 'How deleting the parent document will propagate to child documents.',
```

Better match language we use elsewhere in our documentation.